akka-persistence/build.sbt

```diff
@@ -5,4 +5,6 @@ Formatting.formatSettings
 OSGi.persistence
 Dependencies.persistence
 
+unmanagedSourceDirectories in Compile +=
+  file((sourceDirectory in Compile).value.absolutePath + "/scala-" + scalaVersion.value.dropRight(2))
```

Comment:
I thought sbt did this automatically? 
How does `dropRight(2)` work for both `2.11.10` and `2.12.1`?